spring-cloud-commons/src/main/java/org/springframework/cloud/client/circuitbreaker/CircuitBreakerFactory.java

```diff
@@ -20,10 +20,13 @@
  * Creates circuit breakers based on the underlying implementation.
  *
  * @author Ryan Baxter
+ * @author Andrii Bohutskyi
  */
 public abstract class CircuitBreakerFactory<CONF, CONFB extends ConfigBuilder<CONF>>
 		extends AbstractCircuitBreakerFactory<CONF, CONFB> {
 
 	public abstract CircuitBreaker create(String id);
 
+	public abstract CircuitBreaker create(String id, String groupName);
```

Comment:
Also, it probably can not be abstract (as existing implementations will break). The default implementation should just call the other method.

Reply:
Good point. Changed